src/libraries/Common/tests/SourceGenerators/RoslynTestUtils.cs

```diff
@@ -215,7 +228,8 @@ public static TextSpan MakeSpan(string text, int spanNum)
             string? defaultNamespace = null,
             string? extraFile = null)
         {
-            Project proj = CreateTestProject(references);
+            using var workspace = CreateTestWorkspace();
```

Comment:
For my education, OOM's were triggered by `AdhocWorkspace` instances not being disposed?

Reply:
that is right